src/improvements/tasks/eth/020-U16-remove-dgm/VALIDATION.md

```diff
@@ -0,0 +1,119 @@
+# Validation
+
+This document can be used to validate the inputs and result of the execution of the transaction which you are
+signing.
+
+The steps are:
+
+1. [Validate the Domain and Message Hashes](#expected-domain-and-message-hashes)
+2. [Verifying the state changes via the normalized state diff hash](#normalized-state-diff-hash-attestation)
+3. [Verifying the transaction input](#understanding-task-calldata)
+4. [Verifying the state changes](#task-state-changes)
+
+## Expected Domain and Message Hashes
+
+First, we need to validate the domain and message hashes. These values should match both the values on your ledger and
+the values printed to the terminal when you run the task.
+
+> [!CAUTION]
+>
+> Before signing, ensure the below hashes match what is on your ledger.
+>
+> ### Nested Safe 1 (Foundation): `0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A`
+>
+> - Domain Hash: `0x4e6a6554de0308f5ece8ff736beed8a1b876d16f5c27cac8e466d7de0c703890`
+> - Message Hash: `0x5da3bb2678fc09a51ea7a1449fe2b36c97c80c0c7cfba0669ee0a8997e203731`
+>
+> ### Nested Safe 2 (Security Council): `0xc2819DC788505Aac350142A7A707BF9D03E3Bd03`
+>
+> - Domain Hash: `0xdf53d510b56e539b90b369ef08fce3631020fbf921e3136ea5f8747c20bce967`
+> - Message Hash: `0x14ccefe3aaec9fb93aa84b7ec57c17f7ecdadda8d404eef2a016e102bbe52973`
```

Comment:
Despite this being a nested task, I think that it's only the security council which is a signer on the guardian safe.